layouts/partials/fragments/buttons.html

```diff
@@ -1,23 +1,16 @@
 {{- $self := .self -}}
 {{- $bg := .self.Scratch.Get "bg" }}
 
-{{ "<!-- Buttons -->" | safeHTML }}
-<section id="{{ .Name }}">
-  <div class="overlay container-fluid
-    {{- printf " bg-%s" $bg -}}
-  ">
-    <div class="container py-4">
-      {{- partial "helpers/section-header.html" (dict "bg" $bg "params" .Params) }}
-      <div class="row text-center justify-content-center">
-        {{- range .Params.buttons }}
-          <a class="btn btn-lg m-2{{ if hasPrefix .url "#" }} anchor{{ end }}
-            {{- $color := .color | default "primary" -}}
-            {{- printf " btn-%s" $color -}}
-          " href="{{ .url | relLangURL }}">
-            {{- .text -}}
-          </a>
-        {{- end }}
-      </div>
-    </div>
+{{- partial "helpers/container.html" (dict "start" true "fragment" "Buttons" "in_slot" .in_slot "bg" $bg "Name" .Name "padding" .Params.padding) -}}
```

Comment:
That is used to render this comment: https://github.com/okkur/syna/blob/master/layouts/partials/fragments/buttons.html#L4
It would be useful for debugging the layout if someone wants to debug using source code.

Reply:
Makes sense.